README.md

```
@@ -134,6 +136,46 @@ VITE_GOOGLE_CLIENT_ID="<YOUR_GOOGLE_OAUTH_CLIENT_ID>"
+1. Enable 2-Factor Authentication on your Gmail account.
+2. Go to Google Account settings.
+3. Navigate to Security > 2-Step Verification > App passwords.
+4. Generate a new app password for "Mail".
+5. Use this app password in your SMTP configuration instead of your regular Gmail password.
+
+Example SMTP configuration in config.prod.yml:
+``yaml
+smtp:
+  host: "smtp.gmail.com"
+  port: 587
+  username: "your-email@gmail.com"
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)